Gaming the Syllabus/Assets/Scripts/Portal.cs

```diff
@@ -10,4 +10,8 @@ public class Portal : MonoBehaviour
     void OnTriggerEnter2D(Collider2D Player) {
         SceneManager.LoadScene(levelName); 
     }
+
+    public void buttonLoad() {
```

Comment:
CamelCase is standard for C# so this should be ButtonLoad()

Reply:
Also might be worth considering another name for this since that is more specific to its function!